packages/testing/src/testing-workflow-environment.ts

```diff
@@ -25,6 +25,15 @@ export type TimeSkippingTestWorkflowEnvironmentOptions = {
   client?: ClientOptionsForTestEnv;
 };
 
+/**
+ * Options for {@link TestWorkflowEnvironment.createExistingServer}
+ */
+export type ExistingServerTestWorkflowEnvironmentOptions = {
+  /** If not set, defaults to localhost:7233 */
+  address?: string;
+  client?: ClientOptions;
```

Comment:
Why `ClientOptions` rather than `ClientOptionsForTestEnv` like other variants? I'm particularly worried about the fact that this would make it possible for caller to pass in an existing `Connection`, which would override (line 96) the one we create at line 222.

Reply:
Really just because you need namespace - but I can add that in to the type explicitly